python_bindings/__init__.py

```diff
@@ -0,0 +1,4 @@
+import faulthandler
+from .cpp_bindings import *
```

Comment:
so this is the rexport that makes everything appear under hnswlib?

Reply:
yes